components/twitter/twitter.app.mjs

```diff
@@ -152,21 +151,22 @@ module.exports = {
         });
       },
     },
-    since_id: {
+    sinceId: {
       type: "string",
       label: "Since ID",
       description: "Returns results with an ID greater than (that is, more recent than) the specified ID. There are limits to the number of Tweets which can be accessed through the API. If the limit of Tweets has occured since the since_id, the since_id will be forced to the oldest ID available.",
```

Comment:
```suggestion
      description: "Returns results with an ID greater than (that is, more recent than) the specified ID. There are limits to the number of Tweets that can be accessed through the API. If the limit of Tweets has occurred since the `since_id`, the `since_id` will be forced to the oldest ID available.",
```